File: app/src/androidTest/java/com/przemolab/oknotifier/modules/TestNotifierRepositoryModule.java
package com.przemolab.oknotifier.modules;

import android.content.Context;

import org.mockito.Mockito;

public class TestNotifierRepositoryModule extends NotifierRepositoryModule {

    public TestNotifierRepositoryModule(Context context) {
        super(context);
    }

    @Override
    public NotifierRepository provideNotifierRepository() {
        return Mockito.mock(NotifierRepository.class);
    }
}

File: app/src/androidTest/java/com/przemolab/oknotifier/modules/TestNotifierRepositoryModule.kt
package com.przemolab.oknotifier.modules

import android.content.Context
import com.przemolab.oknotifier.interfaces.INotifierRepository

import org.mockito.Mockito

class TestNotifierRepositoryModule(context: Context) : NotifierRepositoryModule(context) {

    override fun provideNotifierRepository(): INotifierRepository {
        return Mockito.mock(INotifierRepository::class.java)
    }
}
